src/general_manager/manager/generalManager.py

```
@@ -1,5 +1,5 @@
 from __future__ import annotations
-from typing import Generic, Type, Any, TYPE_CHECKING, TypeVar
+from typing import Generic, Type, Any, TYPE_CHECKING, TypeVar, Self
```

Ruff (0.11.9)

`typing.Generic` imported but unused

Remove unused import

(F401)